src/types/auth.ts

```diff
@@ -4,7 +4,8 @@ import { Session } from 'next-auth'
  * Object representing an authentication session.
  */
 type SessionData = Omit<Session, 'user'> & {
-	id?: 'string'
+	accessToken?: string
+	id?: string
```

Comment:
Just curious why these are optional?

Reply:
I can't remember if we saw issues otherwise or not. They may not need to be optional? Hmmm 🤔

Reply:
We could try to make them required! I guess _technically_ there's no guarantee that they will be there as the token could not have the data or something, but in practice it might be okay for them to be required!

Reply:
I'll make a mental note to check this out next week and make the update if possible! 😊